src/app/body/filter-page/filter-page.component.css

```diff
@@ -24,4 +34,4 @@
     background-color: var(--secondary-bg);
     color: var(--text);
     width: 24rem
```

Comment:
do not use rem here, maybe define width in percentage

Reply:
okay I did not make this change but if you say I will add it in my pr